lib/linker.js

```diff
@@ -127,24 +127,6 @@ module.exports = CoreObject.extend({
     return unstable;
   },
 
-  diffGraph: function() {
```

Comment:
We no longer need this because `_updateDepGraphs` is always called and is smart enough to handle idempotent rebuilds and builds where the graph is mutated.